### PR TITLE
chore(lavamoat-node): move lavamoat-node to lavamoat-node directory

### PR DESCRIPTION
**Pull request by boneskull**
This is needed because `@lavamoat/node` is moving in.

### CODE REVIEW

**Comment by boneskull** [comment]
* **#1388** <a href="https://app.graphite.dev/github/pr/LavaMoat/LavaMoat/1388?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @boneskull and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/browserify/test/util.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ const WORKSPACE_ROOT = path.resolve(__dirname, '..', '..', '..')
 const localLavaMoatDeps = {
   lavapack: '@lavamoat/lavapack',
   browserify: 'lavamoat-browserify',
-  node: 'lavamoat',
+  'lavamoat-node': 'lavamoat',
```
Comment (boneskull) [inline packages/browserify/test/util.js:25]: this one was a pain to find 😄

**Review by kumavis — APPROVED** [review]

**Comment by legobeat** [comment]
Could we take this opportunity to make the naming more specific? For example how about `lavamoat-node-runner`?

> This is needed because `@lavamoat/node` is moving in.

Similarly: `@lavamoat/node-runner`? `@lavamoat/node-runner-esm`? or anything to disambiguate.

**Comment by naugtur** [comment]
We do want the external names to be short and friendly and easy to type. 
The internal folder names should preferably be close to `packageName.replace('@lavamoat/','')` and something else if unprefixed

**Pull request merged**